src/java.base/share/classes/java/lang/System.java

```diff
@@ -147,8 +147,7 @@ private System() {
      * corresponds to display output or another output destination
      * specified by the host environment or user. The encoding used
      * in the conversion from characters to bytes is equivalent to
-     * {@link Console#charset()} if the {@code Console} exists,
-     * <a href="#stdout.encoding">stdout.encoding</a> otherwise.
+     * <a href="#stdout.encoding">stdout.encoding</a>.
```

Comment:
```suggestion
     * {@systemProperty stdout.encoding}.
```

This defines the anchor with the same id, and also makes this property available for javadoc index search. Example include the system property at https://download.java.net/java/early_access/jdk24/docs/api/java.base/java/lang/ClassLoader.html#java.system.class.loader

Reply:
Hello Chen, the `{@systemProperty}` tag is expected to be used where the system property is being defined (the "Where should the tag be used?" section in https://mail.openjdk.org/pipermail/core-libs-dev/2018-November/056653.html). 
On this line, it's merely being referenced and the actual definition of this property is in the documentation of System.getProperties() method and there it correctly already uses a `{@systemProperty}` tag for this property https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/lang/System.java#L790

Reply:
Indeed my bad, this is an href instead of an id.